.github/workflows/ci.yml

```diff
@@ -40,8 +40,7 @@ jobs:
     - name: Test with pytest (windows)
       if: startsWith(matrix.os, 'windows')
       run: |
-        .venv\Scripts\activate
-        pytest --xdoc --cov=primap2 --cov-report=xml
+        pytest --cov=primap2 --cov-report=xml
```

Comment:
This is wrong, apparently I misunderstood the merging interface of github, will correct